pkgs/development/web/ping/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkgconfig
+, pantheon
+, python3
+, gtk3
+, gtksourceview
+, json-glib
+, libsoup
+, libgee
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  name = "ping-${version}";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "jeremyvaartjes";
+    repo = "ping";
+    rev = version;
+    sha256 = "1h9cdy2jxa2ffykjg89j21hazls32z9yyv3g0x07x3vizzl5xcij";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pantheon.vala
+    pkgconfig
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtk3
+    gtksourceview
+    json-glib
+    libgee
+    libsoup
+    pantheon.granite
+  ];
+
+  postPatch = ''
+    chmod +x meson/post_install.py
+    patchShebangs meson/post_install.py
+  '';
+
+  meta = with stdenv.lib; {
+    description = " A helpful tool that lets you degug what part of your API is causing you issues";
+    homepage = https://github.com/jeremyvaartjes/ping;
+    maintainers = with maintainers; [ kjuvi ];
```

```suggestion
    maintainers = with maintainers; [ kjuvi ] ++ pantheon.maintainers;
```

Useful for me since I maintain the underlying packaging set.